README.md

```diff
@@ -139,10 +139,8 @@ In addition to code blocks, some code blocks should be rendered differently depe
 ### Example
 
 <pre>
```

Comment:
This suggestion means to revert the deleted line though it looks like nothing have changed when rendered.
```suggestion
<pre>
```
```